client/my-sites/plan-features-comparison/actions.jsx

```diff
@@ -93,6 +135,17 @@ const PlanFeaturesActionsButton = ( {
 	return null;
 };
 
+const PlanFeaturesComparisonActions = ( props ) => {
+	return (
+		<div className="plan-features-comparison__actions">
+			<div className="plan-features-comparison__actions-buttons">
+				<PlanFeaturesActionsButton { ...props } />
+			</div>
+			<PlanFeaturesActionSuggestion { ...props } />
```

Comment:
To avoid errors on `/start/plan`: 
```suggestion
			{ props.isInMarketplace && <PlanFeaturesActionSuggestion { ...props } /> }
```